misaligned.js

```diff
@@ -13,4 +13,6 @@ function print_color_map() {
 
 result = print_color_map();
 expect(result).equals(25);
+expect(result).equals('25'); // need to check the data type of the object 

```

Comment:
One of these expects will fail, irrespective of the implementation.
Use the expects to state the functionality you intend to implement.